android/src/main/java/com/brentvatne/exoplayer/VideoPlaybackService.kt

```diff
@@ -224,7 +226,32 @@ class VideoPlaybackService : MediaSessionService() {
         mediaSessionsList.clear()
     }
 
+    private fun createPlaceholderNotification(): Notification {
+        // Create a notification channel if needed
+        val notificationManager = getSystemService(Context.NOTIFICATION_SERVICE) as NotificationManager
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            notificationManager.createNotificationChannel(
+                NotificationChannel(
+                    NOTIFICATION_CHANEL_ID,
+                    NOTIFICATION_CHANEL_ID,
+                    NotificationManager.IMPORTANCE_LOW
+                )
+            )
+        }
+
+        // Return a minimal notification
+        return NotificationCompat.Builder(this, NOTIFICATION_CHANEL_ID)
+            .setSmallIcon(androidx.media3.session.R.drawable.media3_icon_circular_play)
+            .setContentTitle("Media playback")
```

Comment:
I don't like these hardcoded string ... (As I am french :) ) 
Can you either put empty string of extract to the resources (futur translations will be possible) ?

Reply:
@freeboub thanks for the suggestion @freeboub! it's all set now: https://github.com/TheWidlarzGroup/react-native-video/pull/4453/commits/1c4849331be7e98e06cef932821e0236bf3a59d8